buildspec.yml

```diff
@@ -6,26 +6,12 @@ phases:
         python: 3.7
     commands:
       -  pip install --upgrade 'six==1.15'
+      -  pip install --upgrade 'PyYAML==6.0'
       -  pip install pre-commit cloudformation-cli-java-plugin
   build:
     commands:
       - pre-commit run --all-files
-      - |
-        if [ "$(ls -A $CODEBUILD_SRC_DIR)" ]; then
-          cd $CODEBUILD_SRC_DIR
-          # skip hidden folders
-          dirs=$(find . -not -path "\./\.*" -mindepth 1 -maxdepth 1 -type d)
-          echo "Folders to build: $dirs"
-          for directory in $dirs; do
-            cd "$directory"
-            mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean verify --no-transfer-progress
-            if [ "$?" -ne 0 ] ; then
-                echo "Build failed!"
-                exit 1
-            else
-                cd ".."
-            fi
-          done
-        else
-          echo "$CODEBUILD_SRC_DIR is empty"
-        fi
+      - cd "$CODEBUILD_SRC_DIR/aws-ec2-enclavecertificateiamroleassociation"
+      - mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B clean verify --no-transfer-progress
```

Comment:
I would not change this as the initial version is more generic in case we will add other resource types in the future.